packages/core/package.json

```diff
@@ -54,6 +54,18 @@
   "types": "./dist/type/index.d.ts",
   "main": "./dist/cjs/index.js",
   "exports": {
+    "./readers": {
+      "edge-light": {
+        "types": "./dist/type/readers/index.d.ts",
+        "default": "./dist/readers/index.edge-light.js"
+      }
+    },
+    "./storage": {
+      "edge-light": {
+        "types": "./dist/type/storage/index.d.ts",
+        "default": "./dist/storage/index.edge-light.js"
+      }
+    },
```

Comment:
@himself65 I added edge-specific code, which is transpiled but not used. My nextjs project build still shows this import trace:
```
Import trace for requested module:
../LlamaIndexTS/node_modules/.pnpm/mammoth@1.6.0/node_modules/mammoth/lib/index.js
../LlamaIndexTS/packages/core/dist/readers/DocxReader.js
../LlamaIndexTS/packages/core/dist/readers/index.js
../LlamaIndexTS/packages/core/dist/index.js
```
any idea?

Reply:
yes because you didn't trigger the "subpath exports". You have to manually import from `import 'llamaindex/storage'` for every import path. otherwise it not working

Reply:
So that's why I suggest splitting them into `@llamaindex/xxx`.

Reply:
Another idea is to use [subpath import](https://nodejs.org/api/packages.html#subpath-imports) but it only works in the nodejs latest version